### PR TITLE
Add DeveloperCard component

### PR DESCRIPTION
**Pull request by iceicery**
In this PR, besides create DeveloperCard component, I adjusted two components:
1. Icon: I added background prop at Icon component for adjusting link hover style in light background.
2. Avatar: I adjusted Avatar component in order to show gradient background image.
Please review. Thanks,

### CODE REVIEW

File: src/components/DeveloperCard/DeveloperCard.css [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+  .developercard__avatar {
+    margin: 4px 0 0 19px;
+  }
+
+  .developercard__line {
+    margin: 16px 0 0 0;
+  }
+
+  .developercard__text {
+    margin: 16px 0 10px 0;
```
Comment (julieg18) [inline src/components/DeveloperCard/DeveloperCard.css:133]: The links are slightly too far down, I was able to ammend this by change this line to `margin: 16px 0 8px 0` 
image
image

File: src/components/DeveloperCard/DeveloperCard.css [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+  }
+
+  .developercard__job {
+    margin: 12px 0 0 0;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  .developercard__location {
+    margin: 12px 0 0 10.3px;
```
Comment (julieg18) [inline src/components/DeveloperCard/DeveloperCard.css:119]: I think this should be `margin: 12px 0 0 8px`
image

File: src/components/DeveloperCard/DeveloperCard.css [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+  }
+
+  .developercard__text {
+    margin: 16px 0 10px 0;
+    font-size: 16px;
+    line-height: 28px;
+  }
+}
+
+@media (min-width: 1024px) {
```
Comment (julieg18) [inline src/components/DeveloperCard/DeveloperCard.css:139]: While the card is the right height, some of the elements are a little off. Perhaps you could adjust the margin on the paragraph/divider line... 🤔 
image

File: src/components/DeveloperCard/DeveloperCard.css [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+    font-size: 14px;
+    line-height: 22px;
+  }
+
+  .developercard__location-container {
+    align-items: baseline;
+  }
+
+  .developercard__location {
+    margin: 16px 0 0 10.3px;
```
Comment (julieg18) [inline src/components/DeveloperCard/DeveloperCard.css:166]: As in the other sizes, I think this needs to be `margin: 16px 0 0 8px`.
image

File: src/components/DeveloperCard/DeveloperCard.css [hunk 5]
```diff
@@ -0,0 +1,184 @@
+.developercard {
```
Comment (julieg18) [inline src/components/DeveloperCard/DeveloperCard.css:1]: The card looks awesome!

**Review by julieg18 — CHANGES_REQUESTED** [review]
You've done a great job! Let me know if you have any questions about my comments!

**Review by julieg18 — CHANGES_REQUESTED** [review]
Awesome job! Everything lines up a lot better! Just fix the typo, and we should be able to merge!

File: src/components/Icon/Icon.css [hunk 6]
```diff
@@ -52,7 +58,7 @@
     width: var(--width-l);
   }
 
-  .icon_is-link {
+  .icon_is-link_dark .icon_is-link_light {
```
[the remark this answers is not in the document]
Comment (iceicery) [reply src/components/Icon/Icon.css]: Oh my goodness! Thanks for catching that. What happened to my eyes!!! Fixed!

File: src/components/Icon/Icon.css [hunk 7]
```diff
@@ -46,7 +46,7 @@
     width: var(--width-m);
   }
 
-  .icon_is-link_dark .icon_is-link_light {
+  .icon_is-dark-link .icon_is-light-link {
```
Comment (julieg18) [inline src/components/Icon/Icon.css:49]: I believe this needs a comma: `.icon_is-dark-link, .icon_is-light-link`

**Review by julieg18 — CHANGES_REQUESTED** [review]
Almost done!!!

File: src/components/Icon/Icon.css (hunk 7 above)
Comment (julieg18) [reply src/components/Icon/Icon.css:49]: Same with the other line corrected.

Comment (iceicery) [reply src/components/Icon/Icon.css]: Thanks!!Corrected!

**Review by julieg18 — APPROVED** [review]
Approved!

**Pull request merged**